src/components/autosuggest/autosuggest.ui.js

```diff
@@ -300,7 +300,10 @@ export default class AutosuggestUI {
             this.customResultsThreshold != null && this.customResultsThreshold >= 0 && this.customResultsThreshold <= 1
                 ? this.customResultsThreshold
                 : 0.2;
-        const results = await runFuse(sanitisedQuery, data, this.lang, threshold);
+
+        const distance = threshold >= 0.6 ? 500 : threshold >= 0.4 ? 300 : 100;
```

Comment:
I don't think chaining ternaries together is a good idea, its not very readable. You are probably better off just using if else